_data/trans/sk.json

```diff
@@ -30,12 +30,12 @@
     "sendmail": "Poslať e-mail",
     "showinfo": "ukázať informácie...",
     "submit": "Pridaj stránku",
-    "tagline": "Zoznam linkov na okamžité zmazanie účtov z internetových služieb.",
-    "title": "Just Delete Me | Zoznam linkov na okamžité zmazanie účtov z internetových služieb",
-    "translationcontrib": "Ludia, ktorý pomohli s prekladom",
-    "viewcontrib": "Pozrieť všetkých prispievatelov",
-    "whatisthis": "O čom jr tento projekt?",
-    "whatisthis1": "Väčšina spoločností používajú <a href=\"http://darkpatterns.org/\">dark pattern</a> techniky, aby urobili náročnejšie násť vymazanie účtu . JustDeleteMe sa zameriava na zoznam linkov, ktoré vám umožnia jednoducho vymazať váš účet.",
-    "whatisthis2": "Máš strínku, ktorú by si mohol pridať? <a href=\"https://github.com/jdm-contrib/jdm\">Fork the project on GitHub</a>.",
+    "tagline": "Zoznam odkazov na okamžité zmazanie účtov z internetových služieb.",
+    "title": "Just Delete Me | Zoznam odkazov na okamžité zmazanie účtov z internetových služieb",
+    "translationcontrib": "Ľudia, ktorí pomohli s prekladom",
+    "viewcontrib": "Pozrieť všetkých prispievateľov",
+    "whatisthis": "O čom je tento projekt?",
+    "whatisthis1": "Mnoho spoločností používa <a href=\"http://darkpatterns.org/\">nekalé praktiky</a>, aby urobili vymazanie účtu náročnejšie. JustDeleteMe má za cieľ byť zoznamom odkazov, ktoré vám umožnia jednoducho vymazať váš účet z internetových služieb.",
```

Comment:
According to Google Translate (which I understand is not the authoritative source of translations), this particular change causes the translation to become:

> Many companies use unfair practices to make debiting more challenging. JustDeleteMe aims to be a list of links that will allow you to easily clear your internet service account.

When the original translation was:

> Most companies use dark pattern techniques to make it more difficult to delete the account. JustDeleteMe focuses on a list of links that will allow you to easily delete your account.

I'm ok with the change of "dark pattern" -> "unfair practices" as that is still reasonably well understood, however "debiting" doesn't seem accurate at all, and "easily clear your internet service account" also does not sound correct. If I change the Slovak translation to the following:

> Mnoho spoločností používa nekalé praktiky, aby urobili vymazanie účtu náročnejšie násť . JustDeleteMe má za cieľ byť zoznamom odkazov, ktoré vám umožnia jednoducho vymazať váš účet.

The translation becomes:

> Many companies use unfair practices to make account deletion more difficult. JustDeleteMe aims to be a list of links that will allow you to easily delete your account.

What are your feelings/feedback on that?

P.S. I do not speak Slovak, nor do I claim to have any proficiency in the language; I'm simply tinkering with Google Translate to see how the translations read back in translated English.

Reply:
Well, "unfair practices" is what you would use in Slovak in the meaning of English "dark patterns".

Some English sentences and word composition just doesn't give completely meaningful Slovak translations trough Google Translate. And vice versa. That's why feel not OK when you see the next round of the translation.

BTW., original text is:

> Many companies use dark pattern techniques to make it difficult to find how to delete your account. JustDeleteMe aims to be a directory of urls to enable you to easily delete your account from web services.

That's on the English version of the site.